airwallex/build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
 }
 
 apply from: "$rootProject.projectDir/android-module.gradle"
+apply from: "$rootProject.projectDir/android-shared-compose.gradle"
```

Comment:
This and `card/build.gradle` are probably the only places that need compose dependency for now. We can leave other gradle as it is.